hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/token/impl/CryptoPreTransactionHandlerImpl.java

```diff
@@ -121,7 +121,8 @@ public TransactionMetadata preHandleUpdateAccount(TransactionBody txn) {
     @Override
     /** {@inheritDoc} */
     public TransactionMetadata preHandleCryptoTransfer(TransactionBody txn) {
-        throw new NotImplementedException();
+        final var payer = txn.getTransactionID().getAccountID();
+        return new SigTransactionMetadata(accountStore, txn, payer);
     }
```

Comment:
Should this method be somehow calling `createAccountSigningMetadata`? Can a crypto transfer work with accounts that have receiverSigRequired and we need to check that?